types/node/child_process.d.ts

```diff
@@ -259,7 +259,7 @@ declare module "child_process" {
         cmd?: string;
         killed?: boolean;
         code?: number;
-        signal?: string;
+        signal?: NodeJS.Signals;
```

Comment:
I think `number` is also ok here.

Reply:
@Flarna docs does not eplicitly state that.

Reply:
I found it there: https://nodejs.org/dist/latest-v12.x/docs/api/child_process.html#child_process_subprocess_kill_signal `signal <number> | <string>` . Seems it's allowed everywhere the user passes something in but in events you always get one of the `NodeJS.Signals` strings.